pkg/mutators/podtemplatespec.go

```diff
@@ -25,92 +29,118 @@ func NewPodTemplateSpec(opts ...OptionsFunc) *PodTemplateSpec {
 	return pts
 }
 
-func (p *PodTemplateSpec) Mutate(inputPts corev1.PodTemplateSpec, limitRangeMemory *limitrange.Config) (corev1.PodTemplateSpec, error) {
+func (p *PodTemplateSpec) Mutate(ctx context.Context, inputPts corev1.PodTemplateSpec, limitRangeMemory *limitrange.Config) (corev1.PodTemplateSpec, error) {
 
 	pts := *inputPts.DeepCopy()
 	if limitRangeMemory == nil {
```

Comment:
Do we want to log and return nil on dry-run?

Reply:
Changed. On dry-run returns nil. Added test case.